packages/core/src/fields/button/actions/RunTemplaterFileButtonActionConfig.ts

```diff
@@ -0,0 +1,36 @@
+import type {
+	ButtonClickContext,
+	ButtonConfig,
+	ButtonContext,
+	RunTemplaterFileButtonAction,
+} from 'packages/core/src/config/ButtonConfig';
+import { ButtonActionType } from 'packages/core/src/config/ButtonConfig';
+import { AbstractButtonActionConfig } from 'packages/core/src/fields/button/AbstractButtonActionConfig';
+import type { IPlugin } from 'packages/core/src/IPlugin';
+
+export class RunTemplaterFileButtonActionConfig extends AbstractButtonActionConfig<RunTemplaterFileButtonAction> {
+	constructor(plugin: IPlugin) {
+		super(ButtonActionType.RUN_TEMPLATER_FILE, plugin);
+	}
+
+	async run(
+		_config: ButtonConfig | undefined,
+		action: RunTemplaterFileButtonAction,
+		_filePath: string,
+		_context: ButtonContext,
+		_click: ButtonClickContext,
+	): Promise<void> {
+		await this.plugin.internal.evaluateTemplaterTemplate(action.templateFile, action.templateFile);
```

Comment:
since the return value is not used, this should be prefixed with `void`

Reply:
and the template file path should be passed through `this.plugin.api.buttonActionRunner.resolveFilePath` once. Though I just noticed that the existing code is also not consistent in that regard.